python-regression/tests/features/machine2/config.yml

```diff
@@ -15,8 +15,5 @@ defaults: &transaction_tests_config_files
   java_options: -agentlib:jdwp=transport=dt_socket,server=y,address=8000,suspend=n -javaagent:/opt/jacoco/lib/jacocoagent.jar=destfile=/iri/jacoco.exec,output=file,append=true,dumponexit=true
 
 nodes:
-  nodeA: #name
-    <<: *transaction_tests_config_files
-
-  nodeB:
```

Comment:
You said purely a synactic change in the description.
Why is a node removed?

Reply:
This node is not actually used in the test so it can be removed to save resources.